contents/blog/startup-before-sunset-ai.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Startup Spotlight: How BeforeSunset AI mastered Product Hunt launches"
+date: 2023-08-30
+author: ["joe-martin"]
+showTitle: true
+rootpage: /blog
+sidebar: Blog
+hideAnchor: true
+featuredImage: ../images/blog/posthog-engineering-blog.png
+featuredImageType: full
+category: Startups
+---
+
+> Each month, we speak to a team enrolled in [PostHog for Startups](/startups) about the way they work and the challenges they face. This month we spoke to BeforeSunset AI about their approach to launching new products!
+
+[BeforeSunset AI](https://www.beforesunset.ai/) is an AI-powered daily planner that takes care of your schedule and helps ward off the stress of missed deadlines and forgotten tasks. Founded in November 2022 by entrepreneur and software engineer Elif Duran, BeforeSunset has since grown to a team of 15 and is used by hundreds of teams. 
+
+“The first version of BeforeSunset was just an internal time tracking tool I’d built,” explains Elif. “But when we launched as a separate venture we saw that no other productivity tool asks you the questions needed to plan a day properly. We realized that AI could help if it knew what you were doing, how long it would take, and how long you have available.”
+
+After rebuilding the product around this AI-powered vision, the team wanted to relaunch in earnest — and in June 2023 debuted on Product Hunt, going on to win [Product of the Day, Week, and Month](https://www.producthunt.com/products/beforesunset). Here’s how:
+
+![BeforeSunset AI Launch on Product Hunt](../images/blog/before-sunset-ai.png)
+
+### 1. Create a product launch plan in advance
+
+Unsurprisingly for a productivity startup, it all began with a plan.
+
+“We had a clear plan in our minds for the product,” says Elif. “We wanted to complete the core features, such as to-do listing and time blocking, and we also wanted to build in AI features. In total it took about three months before we were ready to launch to users.”
+
+This development time afforded the growth team time to plan, while also giving them the benefit of knowing how the product would come together. Work was hard — to the point that Elif worked several 40-hour days — but came together smoothly. 
```

Comment:
40-hour days... that is impressive. I assume this isn't 40-hour weeks either, as that would be just a normal working week...

Reply:
```suggestion
This development time afforded the growth team time to plan, while also giving them the benefit of knowing how the product would come together. Work was hard — to the point that Elif worked several 40-hour sprints without sleeping — but came together smoothly. 
```